Probability Mask

Jake-Janssen opened the pull request:
Outputs the confidence score of the highest labeled class - background class 
Min-max normalization values may differ depending on the application

Commit:
# This is a combination of 5 commits.
# The first commit's message is:
added probability mask

# This is the 2nd commit message:

changing imports

# This is the 3rd commit message:

Fixes for Python 2.7

# This is the 4th commit message:

gitignore weight files

# This is the 5th commit message:

Add flag to disable GPU

Commit:
added probability mask

Commit:
changing imports

mpowelson commented:
@Jake-Janssen This branch has conflicts. You need to rebase your changes onto master. Since they are so minimal it might make more sense to just pull master and add your changes.

Also, if you could squash your commits, that would be preferable.

Commit:
output mask now ranges from 0-1 based on the different between background and label confidence.

Commit:
updated output mask to range between 0-1

Add FCN8 Node

Fix bug when annotation fails

Add xml tag to launch file

Turn on dynamic memory allocation on the GPU

This allows GPU memory to dynamically grow. This is a workaround to fix this issue on RTX cards. https://github.com/tensorflow/tensorflow/issues/24496. However, this can be problematic when sharing memory between applications. TODO: Check and see if issue 24496 has been closed, and change this. Note that since Tensorflow 1.15 is the final 1.x release, this might never happen until this code is upgraded to tensorflow 2.x

Correct arguments when GPU is not disabled in test script

FCN8 Node: Add invert results parameter

Fix octomap mesh mask server

Switch octomap mesh mask limits to <= from <

Add launch file that extracts training images from a bag file

Add toggle to enable/disable point cloud annotation and add doxygen

Remove pc_masker and pc_masker_node

These have been replaced by pointcloud annotator

Minor update to readme

Add Bash script that extracts multiple bag files of data

Transform octomap into mesh frame before masking it

Commit:
updated output mask to range between 0-1
